queue_job/jobrunner/runner.py

```diff
@@ -248,11 +210,8 @@ def urlopen():
             # for HTTP Response codes between 400 and 500 or a Server Error
             # for codes between 500 and 600
             response.raise_for_status()
-        except requests.Timeout:
-            set_job_pending()
```

Comment:
A timeout here is normal behaviour, so we don't want to log it as an exception.

Reply:
right :+1: